partition/src/lib.rs

```diff
@@ -99,14 +79,6 @@ pub(crate) static UDP_IO_RX: Lazy<IoReceiver<UdpSocket>> =
 pub(crate) static TCP_IO_RX: Lazy<IoReceiver<TcpStream>> =
     Lazy::new(|| unsafe { IoReceiver::<TcpStream>::from_raw_fd(CONSTANTS.io_fd) });
 
```

Comment:
Informational: It is a shame the socket needs to be connected in the hypervisor. We may think about using something like [slirp](https://wiki.qemu.org/Documentation/Networking) for more flexible networking.